src/main/java/com/groupesan/project/java/scrumsimulator/mainpackage/ui/panels/EditUserStoryForm.java

```diff
@@ -4,6 +4,7 @@
 import com.groupesan.project.java.scrumsimulator.mainpackage.core.UserRolePermissions;
 import com.groupesan.project.java.scrumsimulator.mainpackage.core.UserRoleSingleton;
 import com.groupesan.project.java.scrumsimulator.mainpackage.impl.UserStory;
+import com.groupesan.project.java.scrumsimulator.mainpackage.state.SimulationSingleton;
```

Comment:
Just one small change. Locally my IDE is telling me this import is unused. Just need to delete it